src/api/app/components/workflow_run_row_component.rb

```diff
@@ -11,8 +11,6 @@ def initialize(workflow_run:)
   def hook_action
```

Comment:
`payload['action']` will be `nil` if the webhook event is not for a pull request. In this case, the Bootstrap badge shouldn't be displayed.

Reply:
That's right, but it would show up for unsupported pull request actions, and we don't want that.

Reply:
We can always remove the condition when we properly skip running workflow runs for unsupported actions.

Reply:
> We can always remove the condition when we properly skip running workflow runs for unsupported actions.

Alright, I am fine with this.